frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts

```diff
@@ -82,45 +88,26 @@ describe('Add/remove disks and NICs on respective VM pages', () => {
     VM_ACTIONS_TIMEOUT_SECS,
   );
 
-  it('ID(CNV-1722) NIC cannot be added twice using one net-attach-def', async () => {
-    await vm.navigateToTab(TAB.NetworkInterfaces);
-    if (
-      (await vm.getAttachedNICs()).filter((nic) => nic.name === multusNetworkInterface.name)
-        .length === 0
-    ) {
+  it(
+    'ID(CNV-1722) NIC can be added twice using one net-attach-def',
```

Comment:
in this case, the NIC isn't added twice, only once. The first is NIC using Pod networking, not a bridge added with the NAD. 

```
it(
    'ID(CNV-1722) NIC can be added twice using one net-attach-def',
    async () => {
      const secondMultusInterface = {
        name: `nic2-${testName.slice(-5)}`,
        model: NIC_MODEL.VirtIO,
        mac: getRandomMacAddress(),
        type: NIC_TYPE.bridge,
        network: multusNAD.metadata.name,
      }
      await vm.addNIC(multusNetworkInterface);
      await vm.addNIC(secondMultusInterface)
      expect(await vm.getAttachedNICs()).toContain(multusNetworkInterface);
      expect(await vm.getAttachedNICs()).toContain(secondMultusInterface);

      const vmInterfaces = getInterfaces(vm.getResource());
      expect(
        _.find(vmInterfaces, (o) => o.name === multusNetworkInterface.name),
      ).toBeDefined();
      expect(
        _.find(vmInterfaces, (o) => o.name === secondMultusInterface.name),
      ).toBeDefined();
    },
    VM_ACTIONS_TIMEOUT_SECS,
  );
```

Reply:
thanks, I was thinking the 1st nic is added by the test above, but it also removes the nic in that test. Updated now.